packages/commons-ui-core/src/theme.js

```diff
@@ -0,0 +1,19 @@
+import { red } from "@mui/material/colors";
```

Comment:
No need for theme… All our components will be designed to follow MUI default theme.